getting-started/deployment/aws-ec2.mdx

```diff
@@ -89,9 +89,72 @@ sudo docker compose down && sudo docker compose up -d
 
 ## TLS
 
-For production usage, it’s recommended to enable TLS (HTTPS). You can achieve this by adding an EC2 Load Balancer to serve traffic to your EC2 instance.
+For production usage, it’s recommended to enable TLS (HTTPS). You can achieve this by adding an EC2 Load Balancer to serve traffic to your EC2 instance, as shown in the next section on custom domains.
 
-Without TLS, cloud templates may not load properly when using the Safari browser.
+In addition to the security aspect, enabling TLS also ensures that [workflow templates](/workflow-management/workflow-templates/) load properly in all browsers.
+
+## Accessing OpenOps via a custom domain over TLS (HTTPS)
+
+If you want to access your OpenOps instance via a custom domain (e.g., `https://openops.your-company.com`) instead of its public IP address, you need to set up an AWS Application Load Balancer (ALB) with a valid TLS certificate from AWS Certificate Manager (ACM), then create a new DNS record and update your OpenOps configuration.
```

Comment:
Change the original title from "TLS" to "TLS and a Custom Domain". We can then remove the title here.

```suggestion
To setup a custom domain with TLS, we'll need to set up an AWS Application Load Balancer (ALB) with a valid certificate, generated by AWS Certificate Manager (ACM), create a new DNS record and then update the OpenOps configuration.
```